plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTable.java

```diff
@@ -33,86 +33,223 @@
 import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
 import org.jkiss.dbeaver.model.struct.DBSObject;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
 
 public class CubridTable extends GenericTable
 {
+    private CubridUser owner;
+    private CubridUser oldOwner;
+    private CubridCharset charset;
+    private CubridCollation collation;
+    private Integer autoIncrement;
+    private boolean reuseOID = true;
+
     public CubridTable(
-            GenericStructContainer container,
+            @NotNull GenericStructContainer container,
             @Nullable String tableName,
             @Nullable String tableType,
-            @Nullable JDBCResultSet dbResult)
-    {
+            @Nullable JDBCResultSet dbResult) {
         super(container, tableName, tableType, dbResult);
 
+        String owner_name;
+        String collation_name;
         if (dbResult != null) {
             String type = JDBCUtils.safeGetString(dbResult, CubridConstants.IS_SYSTEM_CLASS);
+            this.reuseOID = (JDBCUtils.safeGetString(dbResult, CubridConstants.REUSE_OID)).equals("YES");
+            owner_name = JDBCUtils.safeGetString(dbResult, CubridConstants.OWNER_NAME);
+            collation_name = JDBCUtils.safeGetString(dbResult, CubridConstants.COLLATION);
+            autoIncrement = JDBCUtils.safeGetInteger(dbResult, CubridConstants.AUTO_INCREMENT_VAL);
             if (type != null) {
                 this.setSystem(type.equals("YES"));
             }
+        } else {
+            owner_name = getDataSource().getContainer().getConnectionConfiguration().getUserName().toUpperCase();

```

Comment:
I do not see any restrictions on creating a Cubrid connection with an empty user name. So, please add a not null condition here.

Reply:
I have fixed this.